Force generating XJC code with specific language

Currently, trying to build Glassfish can fail if other locale is used as the generated source of the XJC can be with an encoding that is not valid UTF-8.

There are two places were it would be required to change: `/nucleus/admin/server-mgmt/pom.xml` and `appserver/appclient/client/acc-config/pom.xml`

```diff
                               <xjc destdir="${dest}"
                                     schema="${project.basedir}/src/main/resources/xsd/schema/stringsubs.xsd"
-                                    package="com.sun.enterprise.admin.servermgmt.xml.stringsubs" />
+                                    package="com.sun.enterprise.admin.servermgmt.xml.stringsubs">
+                                    <jvmarg value="-Duser.language=en" />
+                                    <jvmarg value="-Duser.country=US" />
```

Without the fix

```
[ERROR] /C:/workspace/glassfish/nucleus/admin/server-mgmt/target/generated-sources/xjc/com/sun/enterprise/admin/servermgmt/xml/domaininfo/DomainInfo.java:[4,21] unmappable character (0xE7) for encoding UTF-8
[ERROR] /C:/workspace/glassfish/nucleus/admin/server-mgmt/target/generated-sources/xjc/com/sun/enterprise/admin/servermgmt/xml/domaininfo/DomainInfo.java:[4,22] unmappable character (0xF5) for encoding UTF-8
[ERROR] /C:/workspace/glassfish/nucleus/admin/server-mgmt/target/generated-sources/xjc/com/sun/enterprise/admin/servermgmt/xml/domaininfo/DomainInfo.java:[4,43] unmappable character (0xE3) for encoding UTF-8
[ERROR] /C:/workspace/glassfish/nucleus/admin/server-mgmt/target/generated-sources/xjc/com/sun/enterprise/admin/servermgmt/xml/domaininfo/DomainInfo.java:[4,57] unmappable character (0xF3) for encoding UTF-8
```

Generated source code:

```
//
// Este arquivo foi gerado pela Eclipse Implementation of JAXB, v4.0.5 
// Consulte https://eclipse-ee4j.github.io/jaxb-ri 
// Todas as modifica��es neste arquivo ser�o perdidas ap�s a recompila��o do esquema de origem. 
//
```

--- a/appserver/appclient/client/acc-config/pom.xml
+++ b/appserver/appclient/client/acc-config/pom.xml
@@ -169,8 +169,7 @@
                                     <binding dir="${project.basedir}/src/main/resources" includes="configBindings.xml"/>
                                     <arg value="-dtd"/>
                                     <jvmarg value="-DenableExternalEntityProcessing=true"/>
-                                    <jvmarg value="-Duser.language=en" />
-                                    <jvmarg value="-Duser.country=US" />
+                                    <jvmarg value="-Dfile.encoding=${project.build.sourceEncoding}" />
                                 </xjc>
                             </target>
                         </configuration>

--- a/appserver/appclient/client/acc-config/pom.xml
+++ b/appserver/appclient/client/acc-config/pom.xml
@@ -169,6 +169,8 @@
                                     <binding dir="${project.basedir}/src/main/resources" includes="configBindings.xml"/>
                                     <arg value="-dtd"/>
                                     <jvmarg value="-DenableExternalEntityProcessing=true"/>
+                                    <jvmarg value="-Duser.language=en" />
+                                    <jvmarg value="-Duser.country=US" />
                                 </xjc>
                             </target>
                         </configuration>

--- a/nucleus/admin/server-mgmt/pom.xml
+++ b/nucleus/admin/server-mgmt/pom.xml
@@ -145,13 +145,22 @@
                                 <mkdir dir="${dest}" />
                                 <xjc destdir="${dest}"
                                     schema="${project.basedir}/src/main/resources/xsd/schema/template-info.xsd"
-                                    package="com.sun.enterprise.admin.servermgmt.xml.templateinfo" />
+                                    package="com.sun.enterprise.admin.servermgmt.xml.templateinfo">
+                                    <jvmarg value="-Duser.language=en" />
+                                    <jvmarg value="-Duser.country=US" />
+                                </xjc>
                                 <xjc destdir="${dest}"
                                     schema="${project.basedir}/src/main/resources/xsd/schema/domain-info.xsd"
-                                    package="com.sun.enterprise.admin.servermgmt.xml.domaininfo" />
+                                    package="com.sun.enterprise.admin.servermgmt.xml.domaininfo">
+                                    <jvmarg value="-Duser.language=en" />
+                                    <jvmarg value="-Duser.country=US" />
+                                </xjc>
                                 <xjc destdir="${dest}"
                                     schema="${project.basedir}/src/main/resources/xsd/schema/stringsubs.xsd"
-                                    package="com.sun.enterprise.admin.servermgmt.xml.stringsubs" />
+                                    package="com.sun.enterprise.admin.servermgmt.xml.stringsubs">
+                                    <jvmarg value="-Duser.language=en" />
+                                    <jvmarg value="-Duser.country=US" />
+                                </xjc>
                             </target>
                         </configuration>
                     </execution>

--- a/nucleus/admin/server-mgmt/pom.xml
+++ b/nucleus/admin/server-mgmt/pom.xml
@@ -146,20 +146,17 @@
                                 <xjc destdir="${dest}"
                                     schema="${project.basedir}/src/main/resources/xsd/schema/template-info.xsd"
                                     package="com.sun.enterprise.admin.servermgmt.xml.templateinfo">
-                                    <jvmarg value="-Duser.language=en" />
-                                    <jvmarg value="-Duser.country=US" />
+                                    <jvmarg value="-Dfile.encoding=${project.build.sourceEncoding}" />
                                 </xjc>
                                 <xjc destdir="${dest}"
                                     schema="${project.basedir}/src/main/resources/xsd/schema/domain-info.xsd"
                                     package="com.sun.enterprise.admin.servermgmt.xml.domaininfo">
-                                    <jvmarg value="-Duser.language=en" />
-                                    <jvmarg value="-Duser.country=US" />
+                                    <jvmarg value="-Dfile.encoding=${project.build.sourceEncoding}" />
                                 </xjc>
                                 <xjc destdir="${dest}"
                                     schema="${project.basedir}/src/main/resources/xsd/schema/stringsubs.xsd"
                                     package="com.sun.enterprise.admin.servermgmt.xml.stringsubs">
-                                    <jvmarg value="-Duser.language=en" />
-                                    <jvmarg value="-Duser.country=US" />
+                                    <jvmarg value="-Dfile.encoding=${project.build.sourceEncoding}" />
                                 </xjc>
                             </target>
                         </configuration>

